xml/templates/dao.tpl

```diff
@@ -8,6 +8,11 @@
  * (GenCodeChecksum:{$genCodeChecksum})
  */
 
+
+{if $extension}
+extensionutils
+{/if}
```

Comment:
What does this do?

Reply:
@mlutfy it outputs a string that then CiviX swaps out for a `use CRM_<extesnionname>_Utils as E;` here https://github.com/totten/civix/pull/143/files#diff-b961fdd9b92320d1fd7efa5fd0f49cfeR92 I was struggling to get gencode to respect the spaces between use and the CRM... so did it this way around